lineman/app/models/group_model.coffee

```diff
@@ -17,6 +17,13 @@ angular.module('loomioApp').factory 'GroupModel', (BaseModel, AppConfig) ->
       @hasMany 'subgroups', from: 'groups', with: 'parentId', of: 'id'
       @belongsTo 'parent', from: 'groups'
 
+    proposals: ->
```

Comment:
Pretty sure this wants to be a view, I think you can just go

```
@hasMany 'proposals`, from: 'proposals', with: 'groupId', of: 'id'
```

(You may need to serialize out the groupId in the motionSerializer)